lib/store.js

```diff
@@ -11,48 +11,54 @@ class Store {
       8: 8,
       9: 9,
     },
-    userTeam = null,
-    cpuTeam = null,
+    singlePlayer = null,
+    player1 = {
+      team: null,
+      name: null,
+    },
+    player2 = {
+      team: null,
+      name: null,
```

Comment:
I quite like this way of representing a Player, but I'm a bit concerned that it duplicates information that's also stored in this.singlePlayer (in general, data should be stored in a single canonical place if you can manage it)

What would the store shape look like if we allowed the game to have 0 human players?

My instinct says that we could add a `.isHuman` field to these player objects and get rid of `store.singlePlayer`. (I also made another comment about storing players in an object with "X" and "O" as keys)

Reply:
Logged under https://github.com/declanslevin/o-and-xs/issues/8 to come back to later